hoomd/md/pytest/test_pair.py

```diff
@@ -0,0 +1,939 @@
+                         {"a": [.005, .01, .015], "b": [0.78, 0.04, 0.98]}]
+    forces["Fourier"] = [[-508.812, -5.31354],
+                         [-517.515, -2.42745],
+                         [-527.788, -4.27573]]
+    energies["Fourier"] = [[33.0833, 1.95643],
+                           [35.5141, 1.43308],
+                           [39.5643, 2.47584]]
+
+    params["SLJ"] = [{"sigma": 0.5, "epsilon": 0.0005},
+                     {"sigma": 1.0, "epsilon": 0.001},
+                     {"sigma": 1.5, "epsilon": 0.0015}]
+    forces["SLJ"] = [[390.144, -0.024],
+                     [3220830, -390.144],
+                     [626907000, -76475]]
+    energies["SLJ"] = [[8.064, 0.0],
+                       [67092.5, 16.128],
+                       [13060400, 3184.27]]
+
+    params["DPD"] = [{"A": 0.5, "gamma": 0.0005},
+                     {"A": 1.0, "gamma": 0.001},
+                     {"A": 1.5, "gamma": 0.0015}]
+    forces["DPD"] = [[0.01250824, 0.00714756],
+                     [-0.187336, -0.107049],
+                     [-0.422118, -0.24121]]
+    energies["DPD"] = [[0.30625, 0.1],
+                       [0.6125, 0.2],
+                       [0.91875, 0.3]]
+
+    params["DPDLJ"] = [{'sigma': 0.5, 'epsilon': 0.0005, 'gamma': 0.034},
+                       {'sigma': 1.0, 'epsilon': 0.001, 'gamma': 33.2},
+                       {'sigma': 1.5, 'epsilon': 0.0015, 'gamma': 1.2}]
+    energies["DPDLJ"] = [[-0.000160168, -2.73972 * 10**(-6)],
+                         [0.103803, -0.000320337],
+                         [24.192, 0]]
+    forces["DPDLJ"] = [[2.11492, 1.20788],
+                       [64.2114, 37.7452],
+                       [-377.586, 7.15179]]
+
+    params["DLVO"] = [{'kappa': 1.0, 'Z': 0.1, 'A': 0.1},
+                      {'kappa': 2.0, 'Z': 0.5, 'A': 0.5},
+                      {'kappa': 5.0, 'Z': 2.0, 'A': 2.0}]
+    energies["DLVO"] = [[0.00476409, 0.00226142],
+                        [0.0159279, 0.00357933],
+                        [0.0188554, 0.000449299]]
+    forces["DLVO"] = [[-0.00456658, -0.00226058],
+                      [-0.0309878, -0.00715578],
+                      [-0.0922397, -0.00225115]]
+
+    param_list = []
+    for pair_potential in params.keys():
+        kT_dict = {}
+        if pair_potential == "DPD":
+            kT_dict = {"kT": 2}
+        elif pair_potential == "DPDLJ":
+            kT_dict = {"kT": 1}
+        for i in range(3):
+            param_list.append((pair_potential,
+                               getattr(hoomd.md.pair, pair_potential),
+                               kT_dict,
+                               params[pair_potential][i],
+                               forces[pair_potential][i],
+                               energies[pair_potential][i]))
+    return param_list
+
+
+@pytest.fixture(scope="function",
+                params=_forces_and_energies(),
+                ids=(lambda x: x[0]))
```

Comment:
We will have 3 tests with the same name given this. I think having a number to signify which iteration would be helpful.

Reply:
A number will be append by pytest automatically